packages/core/src/analytics/integrations/__tests__/Integration.test.js

```diff
@@ -29,7 +25,60 @@ describe('Integration', () => {
     expect(integration.track).toBeInstanceOf(Function);
   });
 
-  it('Should have a setConsent method', () => {
-    expect(integration.setConsent).toBeInstanceOf(Function);
+  it('should return an instance of itself or of an extended class', () => {
+    class MyIntegration extends Integration {}
+
+    expect(integration).toBeInstanceOf(Integration);
+    expect(new MyIntegration()).toBeInstanceOf(Integration);
```

Comment:
I think I do not understand the goal of this test. Like this we are testing the Javascript inheritance mechanism, there isn't a particular behavior that is related to analytics/Integration. Perhaps you meant to test the `createInstance` method?

Reply:
yes, it was indeed 😄 fixed, thanks for the heads up!